<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Preview">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>570</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Preview&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">pushButton_Fillbucket:focus {
  border: 1px solid blue !important;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/icon_Preview.png</normaloff>:/images/icon_Preview.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ColorPalette">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Color Palette&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">pushButton_Fillbucket:focus {
  border: 1px solid blue !important;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/icon_ColorPalette.png</normaloff>:/images/icon_ColorPalette.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DeleteFrame">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>570</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete Frame&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">pushButton_Fillbucket:focus {
  border: 1px solid blue !important;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/icon_DeleteFrame.png</normaloff>:/images/icon_DeleteFrame.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AddFrame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>570</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add Frame&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">pushButton_Fillbucket:focus {
  border: 1px solid blue !important;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/icon_AddFrame.png</normaloff>:/images/icon_AddFrame.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">pushButton_Fillbucket:focus {
  border: 1px solid blue !important;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/icon_Pen.png</normaloff>:/images/icon_Pen.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Eraser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Eraser&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">pushButton_Fillbucket:focus {
  outline: 1px solid blue !important;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/icon_Eraser.png</normaloff>:/images/icon_Eraser.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>72</width>
      <height>72</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Fillbucket">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fill Bucket&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">pushButton_Fillbucket:focus {
  border: 1px solid blue !important;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/Icon_fill.png</normaloff>:/images/Icon_fill.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>72</width>
      <height>72</height>
     </size>
    </property>
   </widget>
   <widget class="Frame" name="frameWidget" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_color">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Current Color Selected&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">pushButton_Fillbucket:focus {
  border: 1px solid blue !important;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>570</y>
      <width>511</width>
      <height>91</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>509</width>
       <height>89</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="Frame" name="framesWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>pushButton_Preview</zorder>
   <zorder>pushButton_ColorPalette</zorder>
   <zorder>pushButton_Pen</zorder>
   <zorder>pushButton_Eraser</zorder>
   <zorder>pushButton_Fillbucket</zorder>
   <zorder>pushButton_AddFrame</zorder>
   <zorder>pushButton_DeleteFrame</zorder>
   <zorder>frameWidget</zorder>
   <zorder>pushButton_color</zorder>
   <zorder>scrollArea</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="maximumSize">
     <size>
      <width>298</width>
      <height>400</height>
     </size>
    </property>
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="action_New"/>
    <addaction name="actionSave_As"/>
    <addaction name="action_Save"/>
    <addaction name="action_Load"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>frame</class>
   <extends>QWidget</extends>
   <header>frame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Frame</class>
   <extends>QWidget</extends>
   <header location="global">frame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
